src/Microsoft.DotNet.Cli.Utils/CommandResolver.cs

```diff
@@ -172,12 +172,14 @@ private static CommandSpec ConfigureCommandFromPackage(string commandName, strin
 
                 fileName = CoreHost.Path;
 
+                var depsArg = string.Empty;
+
                 if (depsPath != null)
                 {
-                    args = $"--tpaFile:\"{depsPath}\" {args}";
+                    depsArg = $"\"--depsFile:{depsPath}\" ";
                 }
 
-                args = $"\"{dllPath}\" {args}";
+                args = $"\"{dllPath}\" {depsArg}{args}";
             }
             else
             {
```

Comment:
Thanks @schellap, the implementation became much cleaner with your feedback.

I've gone ahead and made your suggested changes and updated the .md file to match.